recipe/meta.yaml

```
@@ -165,78 +170,76 @@ outputs:
         - cythonize -i -3 test_cimport.pyx
         - python -c "import test_cimport; test_cimport.test()"
         - python -c "import cuda.bindings; print(cuda.bindings.__version__)"
-        - python -c "import cuda; print(cuda.__version__)"  # TODO: Remove this
       imports:
         - cuda
         - cuda.cuda
         - cuda.cudart
     about:
       home: https://nvidia.github.io/cuda-python/cuda-bindings
       license: LicenseRef-NVIDIA-SOFTWARE-LICENSE
-      license_file: LICENSE
-      license_url: https://github.com/NVIDIA/cuda-python/blob/main/LICENSE
-      summary: CUDA Python Low-level Bindings
+      license_file: cuda_bindings/LICENSE
+      license_url: https://github.com/NVIDIA/cuda-python/blob/main/cuda_bindings/LICENSE
+      summary: Low-level Python bindings for CUDA
       description: |
-        CUDA Python provides a standard set of low-level interfaces,
+        cuda.bindings is a standard set of low-level interfaces,
         providing full coverage of and access to the CUDA host APIs from Python.
       doc_url: https://nvidia.github.io/cuda-python/cuda-bindings
       dev_url: https://github.com/NVIDIA/cuda-python
 
   - name: cuda-core
-    version: {{ core_ver }}
+    version: {{ core_version }}
+    # Keep scripts under outputs section to ensure building the modules/packages provided by cuda-python
     script: build_output.sh   # [linux]
     script: build_output.bat  # [win]
     build:
       number: {{ number }}
-      # While building cuda-core does not require CUDA/NVCC, we use the same skip condition here
-      # so that all packages can be built in the same CI pipelines.
-      {% if not (environ.get("cuda_compiler_version")|string()) == (bindings_major_version ~ "." ~ bindings_minor_version) %}
-      skip: true
-      {% else %}
-      skip: true  # [not build_core]
-      {% endif %}
+      skip: {{not build_core}}
+      run_exports:
+        - {{ pin_subpackage('cuda-core', min_pin='x.x', max_pin='x.x') }}
+      ignore_prefix_files:
+        - '**/*.c'
+        - '**/*.cpp'
+        - '**/*.so.c'
     requirements:
       build:
+        - {{ stdlib('c') }}
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
         - {{ compiler('cuda') }}
-        - {{ stdlib('c') }}
-        - cross-python_{{ target_platform }}  # [build_platform != target_platform]
-        - python                              # [build_platform != target_platform]
-        - cython                              # [build_platform != target_platform]
       host:
-        - cython
-        - pip
         - python
+        - pip
         - setuptools
+        - cython
```

```suggestion
        - cython >=3.0
```
https://github.com/NVIDIA/cuda-python/blob/v12.8.0/cuda_core/pyproject.toml#L6